packages/fxa-auth-server/fxa-oauth-server/lib/metrics/amplitude.js

```diff
@@ -79,8 +77,12 @@ module.exports = (log, config) => {
       eventData
     );
 
-    if (sane(amplitudeEvent)) {
-      log.info('amplitudeEvent', amplitudeEvent);
+    if (amplitudeEvent) {
+      if (shouldExclude(amplitudeEvent)) {
+        log.info('excludedAmplitudeEvent', amplitudeEvent);
```

Comment:
I think we'll need to modify the `op` key on the object here too :/

Or we need to switch the Amplitude processing over to GCP entirely, where we can filter out events again